scripts/sync.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Mmmm well I do kill all instances before the sync.sh is run again on cron. Think a lock file is still needed ?